src/js/step.js

```diff
@@ -290,7 +288,11 @@ export class Step extends Evented {
    * @return {boolean} True if the step is open and visible
    */
   isOpen() {
-    return Boolean(this.el && !this.el.hidden);
+    return Boolean(
+      this.tooltip &&
```

Comment:
Just to confirm, we're just using built in Tippy stuff here now, instead of `hidden` right?

Reply:
👍